js-exercises/aperture/aperture.js

```diff
@@ -0,0 +1,13 @@
+function aperture(tuple, array) {
+  if (array.length < tuple) return [];
+  if (tuple == 0) return array;
+  let aperture = [];
+  for (let i = 0; i < (array.length - (tuple - 1)); i++) {
+    aperture.push(array.slice(i, i + tuple));
+  }
+  return aperture;
+}
+const sevenLs = [1, 2, 3, 4, 5, 6, 7];
```

Comment:
this seems like test data please keep in mind not to push console.logs and test data as part of actual code.